ui/library/src/context/ScrollContext.ts

```diff
@@ -216,6 +223,21 @@ export function useScrollContextProps(): IScrollContext {
     };
   }, [scrollRoot, observerIndex]);
 
+  const recalibrateScrollPosition = React.useCallback(
+    (zoomMultiplier: number): void => {
+      // fix known react-pdf bug where zooming causes scroll position to jump to different pages
+      const root = scrollRoot || document.documentElement;
+      if (!root) {
+        return;
+      }
+      const newScrollTop = Math.floor(root.scrollTop * zoomMultiplier);
+      setTimeout(() => {
+        root.scrollTop = newScrollTop;
+      }, 1);
```

Comment:
@ericmarsh995 is there a need for setting 1 for timeout since 1 milisecond doesnt seem that to be alot either we use 0 or 1000 to at least see its slowly for verification